test/tests/functional/pbs_hook_exechost_periodic.py

```diff
@@ -122,21 +118,18 @@ def test_exechost_periodic_accept(self):
         common_msg = " as it is owned by a different mom"
         common_msg2 = "resources_available.mem=9gb per mom hook request"
 
-        msg1 = "%s;Not allowed to update vnode '%s'," % (self.momA.hostname,
-                                                         self.hostB)
-        exp_msg1 = msg1 + common_msg
-        exp_msg2 = "autocreated vnode %s" % other_node
+        exp_msg1 = "autocreated vnode %s" % other_node
+        msg1 = "%s;Updated vnode %s's resource " % (self.momA.hostname,
+                                                    other_node)
+        exp_msg2 = msg1 + common_msg2
         msg2 = "%s;Updated vnode %s's resource " % (self.momA.hostname,
                                                     other_node)
         exp_msg3 = msg2 + common_msg2
```

Comment:
I'm seeing msg1 and msg2 to be the same, so exp_msg2 and exp_msg3 are also the same.